turbo/execution/eth1/block_building.go

```diff
@@ -42,6 +42,7 @@ func (e *EthereumExecutionModule) evictOldBuilders() {
 // Missing: NewPayload, AssembleBlock
 func (e *EthereumExecutionModule) AssembleBlock(ctx context.Context, req *execution.AssembleBlockRequest) (*execution.AssembleBlockResponse, error) {
 	if !e.semaphore.TryAcquire(1) {
+		e.logger.Warn("ethereumExecutionModule.AssembleBlock: ExecutionStatus_Busy")
```

Comment:
I'm not sure how often CL getting it on non-chain-tip (where long rwtx happen). 

Erigon currently using next logs formatting: `[exec] AssembleBlock: ExecutionStatus_Busy`

Reply:
I'm using the format that was used by this class EthereumExecutionModule before my changes.
Nevertheless, it is still inconsistent in some places.
Let's discuss the format separately and I can make a new PR with the log format changes.